docs/getting-started-guides/scratch.md

```diff
@@ -167,6 +167,8 @@ You can use a Kubernetes binary release (recommended) or build your Kubernetes b
 [Developer Documentation](https://github.com/kubernetes/kubernetes/tree/{{page.githubbranch}}/docs/devel/).  Only using a binary release is covered in this guide.
 
 Download the [latest binary release](https://github.com/kubernetes/kubernetes/releases/latest) and unzip it.
+As server binary tarballs are no longer included in the Kubernetes final tarball. You need to locate and run
```

Comment:
Suggested edit: "Server binary tarballs are no longer included in the Kubernetes final tarball, so you will need to locate and run.."